APIs_Platform/FileHandle_sigio/main.cpp

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2006-2020 Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "mbed.h"
+
+static DigitalOut led1(LED1);
+static DigitalOut led2(LED2);
+
+static BufferedSerial device(STDIO_UART_TX, STDIO_UART_RX);
+
+static void callback_ex()
+{
+    // always read until data is exhausted - we may not get another
+    // sigio otherwise
+    while (1) {
+        char c;
+        if (device.read(&c, 1) != 1) {
+            break;
+        }
+        putchar(c);
+        led2 = !led2;
+    }
+}
+
+int main()
+{
+    // BufferedSerial-specific method - all others are from FileHandle base class
+    device.set_baud(19200);

```

Comment:
Changing the baud rate is not useful for this example.

Reply:
Removed and updated `README.md`.